lib/styles/index.scss

```diff
@@ -1,6 +1,6 @@
 /* Global Styles */
-@import "./reset.scss";
 @import "@deriv-com/quill-tokens/dist/quill.css";
+@import "../../node_modules/modern-normalize/modern-normalize.css";
```

Comment:
Maybe we can import the unpkg or CDN version of this instead?

Reply:
Sure I'll use unpkg for this